cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -451,3 +451,38 @@ def test_v2v_with_no_providers(appliance, source_provider, provider, soft_assert
         source_provider.create(validate_inventory=True)
     if is_target_provider_deleted:
         provider.create(validate_inventory=True)
+
+
+@pytest.mark.tier(2)
+def test_duplicate_plan_name(appliance, mapping_data_vm_obj_mini, provider):
+    """
+    Test Migration plan with duplicate names
+
+    Polarion:
+        assignee: sshveta
+        initialEstimate: 1/4h
+        caseimportance: medium
+        caseposneg: negative
+        testtype: functional
+        startsin: 5.10
+        casecomponent: V2V
+        testSteps:
+            1. Create two migration plans with same name
+        expectedResults:
+            1. Plan with duplicate name shows error message
+    """
+    migration_plan_collection = appliance.collections.v2v_migration_plans
+    name = "plan_{}".format(fauxfactory.gen_alphanumeric())
+    migration_plan_collection.create(
+        name=name,
+        description="desc_{}".format(fauxfactory.gen_alphanumeric()),
+        infra_map=mapping_data_vm_obj_mini.infra_mapping_data.get("name"),
+        target_provider=provider,
+        vm_list=mapping_data_vm_obj_mini.vm_list,
+        start_migration=False
+    )
+    view = navigate_to(migration_plan_collection, "Add")
+    view.general.infra_map.fill(mapping_data_vm_obj_mini.infra_mapping_data.get("name"))
+    view.general.name.fill(name)
+    view.general.description.fill("description")
+    assert view.general.alert.read() == "Name {name} already exists".format(name=name)
```

Comment:
Will the browser be stuck on the `Add` page after the test ends? I.e. is it possible to navigate away from this page.